requirements.txt

```diff
@@ -5,3 +5,4 @@ scipy
 matplotlib
 ninja
 h5py >= 3.1.0
+lava-nc@git+https://github.com/lava-nc/lava.git
```

Comment:
Before we release lava-dl we should update this to the released lava version that supports netx.

Reply:
This can happen in release PR after this is merged.